### PR TITLE
Release 1.26.6

### PR DESCRIPTION
**Pull request by pquentin**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,17 @@
 Changes
 =======
 
+1.26.6 (2021-06-25)
+-------------------
+
+* Deprecate the urllib3.contrib.ntlmpool module. urllib3 is not able
```
Comment (sethmlarson) [inline CHANGES.rst:7]: Can we use double-backticks to format the code sections:

- urllib3.contrib.ntlmpool
- Transfer-Encoding
- request_chunked()
- Retry.DEFAULT_ALLOWED_METHODS

**Review by sethmlarson — CHANGES_REQUESTED** [review]

File: CHANGES.rst (hunk 1 above)
Comment (pquentin) [reply CHANGES.rst]: Thanks, fixed

**Review by pquentin — COMMENTED** [review]
Can't approve my pull request, but the rewording is awesome, thanks!

**Review by sethmlarson — APPROVED** [review]
LGTM

**Pull request merged**